docs/source/tutorial.rst

```diff
@@ -760,11 +760,11 @@ restricts parallelism. Of course it would be nice if Python had better
 options for taking advantage of multiple cores, but that's an
 extremely difficult problem to solve, and in the meantime there are
 lots of problems where a single core is totally adequate – or where if
-it isn't, then process- or machine-level parallelism works fine.
+it isn't, then process or machine-level parallelism works fine.
```

Comment:
The intent here is "process-level or machine-level parallelism", so removing the dash changes the meaning here.